examples/amp-story/amp-story-animation.html

```diff
@@ -38,7 +38,7 @@
             fly-in with animate-in, after the previous element
           </div>
         </amp-story-grid-layer>
-        <amp-story-animation layout="nodisplay">
```

Comment:
`layout` is not necessary since these don't have a custom element implementation.

Reply:
It might be still necessary. Do we want an extra inline element floating in the DOM otherwise?

Reply:
I did not realize we had a CSS rule for `[layout=nodisplay]:not(.i-amphtml-element)`. Cool, added it back!